src/slurm_plugin/cluster_event_publisher.py

```diff
@@ -100,6 +100,14 @@
     "message": "Count of EC2 instances backing the cluster compute fleet",
     "event_type": "cluster-instance-count",
 }
+SCONTROL_SHOW_JOB_INFORMATION = {
+    "message": "Job information from scontrol",
+    "event_type": "scontrol-show-job-information",
+}
+SCONTROL_SHOW_JOB_COUNT = {
+    "message": "Count of the jobs from scontrol",
+    "event_type": "scontrol_show_job_count",
```

Comment:
Same as above for this - maybe keep the name more generic rather than mechanism specific. Also replace `_` with `-`

Reply:
Good idea, changed